community/CloudflareR2.md

```diff
@@ -0,0 +1,144 @@
+   1. Give the rule a name, e.g. `BlueMap: Content-Encoding`.
+   2. Select `Custom filter expression`
+      1. In the first field, select `URI Path`. In the Operator field, select `ends with`. In the value field, enter `.gz`.
+      2. Click on `Or` to add another rule.
+      3. In the first field, select `URI Path`. In the Operator field, select `ends with`. In the value field, enter `textures.json`.
+      4. Click on `Or` to add another rule.
+      5.  In the first field, select `URI Path`. In the Operator field, select `ends with`. In the value field, enter `.prbm`.
+   3. Further down under `Modify response header`.
+      1. In the first field, select `Set static`. In the Header name field, enter `Content-Encoding`. In the Header value field, enter `gzip`.
+   4. It should look similar to this: ![Screenshot of the Content-Encoding rule in Cloudflare]({{site.baseurl}}/assets/r2/encoding_rule.png)
+   5. Save the rule. This rule marks the compressed files as compressed, so your browser knows how to handle them when requested.
+
+
+## Making optional configurations
+
+Every time a file is requested from your bucket, that counts as an operation. Every operation above the free limit induces a cost. To reduce the number of requests to your bucket, you can set up a cache policy. This will cache the files on Cloudflare's edge servers and serve them from there. This will reduce the number of requests to your bucket and thus reduce the costs.
+
+1. Go to your domain inside the Cloudflare dashboard. Click on “Rules” and then “Cache Rules”. Click on “Create Rule”.
+   1. Give the Rule a name, e.g. `BlueMap: Cache`.
+   2. Select `Custom filter expression`
+      1. In the first field, select `Hostname`. In the Operator field, select `equals`. In the value field, enter the domain you are using for BlueMap, e.g. `map.example.com`.
+   3. Further down under `Cache eligibility` select `Eligible for cache`.
+   4. Further down under `Edge TTL` click on `Add Setting`.
+      1. Now select `Ignore cache-control header and use this TTL` and set `Input time-to-live (TTL)` to a value you are comfortable with. I set it to 1 year.
+      2. This rule will cache all files for up to 1 year on Cloudflare's edge servers and serve them from there instead of requesting them from your bucket. This will reduce the number of requests to your bucket and thus reduce the costs.
+   5. Further down under `Browser TTL` click on `Add Setting`.
+      1. Now select `Override origin and use this TTL` and set `Input time-to-live (TTL)` to a value you are comfortable with. I set it to 1 year.
+      2. This rule will cache all files for up to 1 year in your browser. This can be beneficial if you have many visitors who visit the map multiple times. They can load the map faster because the files are already in their browser cache. If they want to see the latest version of the map, they can do a hard refresh (Ctrl + F5) to force the browser to load the files from the server.
```

Comment:
This rule was already introduced beforehand, which is good, but the way this is worded does make it seem like you're introducing it again

Reply:
I am not sure what change you are hoping to see here😅

Reply:
Fair enough.
My thought was that this is less of a "step to take", and more of an explanation of what this section is for, like some of the other comments I wrote earlier.
I'd move it to line 132 instead of here.

Reply:
Changed with the latest commit. How do you like this one @TechnicJelle? :)

Reply:
I may be blind, or just bad at navigating the mobile GitHub website, but I'm not seeing something like this in the latest commit, 804ca8d

Reply:
Sorry had a long day... comitted it and then forgot to push. https://github.com/BlueMap-Minecraft/BlueMapWiki/pull/88/commits/3f819063f590f0864811db23900a420807a8cc51

Reply:
Long day for you too, eh?
Same here...
So I'll have a look soon™

Reply:
Yeah it looks good now! :D